tests/test_webdav.py

```diff
@@ -62,6 +62,50 @@ def test_upload_download_file(self):
         self.nxc_local.delete_path(self.user_username, file_name)
         os.remove(file_local_path)
 
+    def test_upload_download_file_with_timestamp(self):
+        file_name = "test_file"
+        file_content = "test file content"
+        file_local_path = os.path.join(os.getcwd(), file_name)
+
+        # 2001-09-09T01:46:40 (UTC & GMT)
+        timestamp = 1000000000
```

Comment:
Please define the timestamp using the literal `"Sun, 09 Sep 2001 01:46:40 GMT"` (and add an assert that it is equal to 1000000000 if you wish).